### ISSUE TITLE
fine-tune failure

### ISSUE THREAD

**Issue by gzp0201** [lyndonzheng/Pluralistic-Inpainting#43]
Thank you for your work! I am trying to fine-tune on places365 on the basis of your trained model 'place2_random'. However, the inpainted result is not as good as yours. I set the relative training arguments as default, and I have trained the network for 177420 iterations with batch size of 128. What else should be taken into account? Thank you. The 1st pic is of your result, 2nd is of mine.
![lawn01](https://user-images.githubusercontent.com/40631498/71789662-53910880-3067-11ea-9ed5-d9c066e35663.png)

![lawn01](https://user-images.githubusercontent.com/40631498/71789586-ee3d1780-3066-11ea-934b-74fbb4d79778.png)

**Comment by lyndonzheng** [comment]
@gzp0201 Please provide more training details for the fine-tuning. Have you tried to test the earlier fine-tuned models, e.g. 1000 iterations? In this way, you may analyze the problem.

**Comment by gzp0201** [comment]
Thank you for your reply. I concatenate all the results into an image. 
![im_002](https://user-images.githubusercontent.com/40631498/71795165-d6be5880-307f-11ea-93ec-4f1dcab8ecef.png)

=================== base_options.py ===================
![image](https://user-images.githubusercontent.com/40631498/71795263-54826400-3080-11ea-823e-0d79baa41c3f.png)
=================== train_options.py ===================
![image](https://user-images.githubusercontent.com/40631498/71795294-85fb2f80-3080-11ea-9ded-9949b46d60f6.png)

**Comment by lyndonzheng** [comment]
@gzp0201 If you fine-tuning the model, you should get results similar to our model at the beginning. The presented results are blurry. I guess the discriminator loss may converge too fast. Besides, your training batch size is 128. I have not tried such a large batch size for the limited GPU memory.

**Comment by gzp0201** [comment]
OK, thank you for your advice. I will recheck it.
